File: 4.1 C Programming/Homework 5 Conditional Statements/DigitAsWord.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char** argv) 
{
    int n;
    scanf("%d", &n);
    switch (n) {
        case 0:
            printf("zero\n");
            break;
            case 1:
            printf("one\n");
            break;
            case 2:
            printf("two\n");
            break;
            case 3:
            printf("three\n");
            break;
            case 4:
            printf("four\n");
            break;
            case 5:
            printf("five\n");
            break;
            case 6:
            printf("six\n");
            break;
            case 7:
            printf("seven\n");
            break;
            case 8:
            printf("eight\n");
            break;
            case 9:
            printf("nine\n");
            break;
        default:
            printf("Not a digit\n");
            break;
    }
    return (EXIT_SUCCESS);
}

File: 0.1 Programming basics/Homework 5 Conditional-Statements/Problem 8. Digit as Word/DigitAsWord.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problem_8.Digit_as_Word
{
    class DigitAsWord
    {
        static void Main(string[] args)
        {
            string n = (Console.ReadLine());
            switch (n)
            {
                case "1":
                    Console.WriteLine("One");
                    break;
                case "2":
                    Console.WriteLine("Two");
                    break;
                case "3":
                    Console.WriteLine("Three");
                    break;
                case "4":
                    Console.WriteLine("Four");
                    break;
                case "5":
                    Console.WriteLine("Five");
                    break;
                case "6":
                    Console.WriteLine("Six");
                    break;
                case "7":
                    Console.WriteLine("Seven");
                    break;
                case "8":
                    Console.WriteLine("Eight");
                    break;
                case "9":
                    Console.WriteLine("Nine");
                    break;
                case "0":
                    Console.WriteLine("Zero");
                    break;
                default:
                    Console.WriteLine("Not a digit");
                    break;
            }
        }
    }
}
